topics/metagenomics/tutorials/pathogen-detection-from-nanopore-foodborne-data/tutorial.md

```diff
@@ -575,7 +552,7 @@ To identify VF or AMR genes, it is better to assemble reads into longer seuqence
     > <hands-on-title> Assembly with Flye </hands-on-title>
     >
     > 1. {% tool [Flye](toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.9+galaxy0) %} with the following parameters:
-    >    - {% icon param-file %} *"Input reads"*: Outputs without matched ID of **Filter sequences by ID** {% icon tool %} for `Barcode10`
+    >    - {% icon param-file %} *"Input reads"*: collection output from **Krakentools: Extract Kraken Reads By ID** {% icon tool %} from the preprocessing section
     >
     >      > <comment-title></comment-title>
     >      > We need to run **Flye** individually on each sample otherwise **Flye** runs by default a co-assembly mode, *i.e.* it combines read of both samples together before running the assembly.
```

Comment:
```suggestion
    >      > We need to run **Flye** individually on each sample otherwise **Flye** runs by default a co-assembly mode, *i.e.* it combines reads of both samples together before running the assembly.
```